src/main/java/seedu/modtrek/logic/commands/AddCommand.java

```diff
@@ -20,41 +20,41 @@ public class AddCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
```

Comment:
Message String should change, think you missed these.
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a module to the degree progression. "
```

Reply:
Thanks for spotting that. I changed that in a few spots.